drivers/src/stm32f446xx_usart_driver_course.c

```diff
@@ -0,0 +1,112 @@
+/*
+ * stm32f446xx_usart_driver_course.c
+ *
+ *  Created on: Apr 24, 2020
+ *      Author: quentin
+ */
+
+#include "stm32f446xx_usart_driver_course.h"
+
+/* USART Initialization */
+void USART_Initialize(USART_Handler_TypedDef* pUSART_Handler)
+{
+	//baud rate equation : txrx baud = Fclk/(8*(2-oversampling)*USARTDIV)
+	float USARTDIV, SysClock;
+
+//	USARTDIV = f/((8*2-pUSART_Handler->USART_Pin_Config.USART_OverSampling)*pUSART_Handler->USART_Pin_Config.USART_BaudRate);
+
```

Comment:
Create a separate function for the Baud Rate, this is more complex than looks like.
`Status_t UART_SetBaudRate(UART_Handle_t *pUARTHandle)`

Review my code https://github.com/Fernandoks/FirstDriver/blob/master/drivers/src/stm32f446xx_usart.c